### ISSUE TITLE
ENH: Transpile to R/Python the Effective Competition Panel Estimation

### ISSUE THREAD

**Issue by csmcallister** [GSA/contracting-hypotheses#3]
## User story 
As a data scientist, I want to replicate the panel analysis done in EViews so that the team's efforts are transparent and replicable.
  
## Acceptance criteria
- [x] A Jupyter Notebook replicating the analysis with documentation in markdown

Included in the markdown, there are notes documenting:
- [x] the data source
- [x] the unit of analysis
- [x] decisions made to prep/clean the data, including how missing values and/or data quality issues were addressed
- [x] reasoning behind the model selection, including the statistical tests used to validate the model's assumptions
- [x] interpretation of the model summary

**Comment by csmcallister** [comment]
The following should provide *most* of what's needed to transpile the analysis into R or Python.

## Summary of the analysis
<img width="624" alt="screen shot 2019-03-08 at 8 11 46 am" src="https://user-images.githubusercontent.com/33464354/54030894-0c9e6200-417b-11e9-9329-d6ffb84f8c10.png">

## Variable Description
The metrics are defined [here](https://drive.google.com/open?id=1OGuhgsoNUeDhK685-Lyv0n4-cuAs6kPT), but there's also this screenshot from the presentation:
<img width="602" alt="screen shot 2019-03-08 at 8 11 41 am" src="https://user-images.githubusercontent.com/33464354/54030895-0c9e6200-417b-11e9-90ee-e48e21723579.png">


## Source Data
The source data for the analysis can is the view `OMB_ACQ_Hypotesting_Rates_vw` in our database.

I'm pretty sure it can also be found [here](https://drive.google.com/open?id=1egy7__YUCXKuDOAH0gd9yddvSz4pAS-z). Note: you'll need to be on the MD team drive in order to access this.

## Model Summary
There are discrepancies between column names in the excel file and column names in the model summary (see screenshot).

Based on the screenshot below, the model is:

```
EFFCOMPSN ~ C + NOTCOMPRATESN(-1) + ONEBIDRATESN + SDBRATESN
```
>The `-1` indicates a one-period lag.
>`C` indicates the intercept according to Ross.

<img width="535" alt="screen shot 2019-03-08 at 8 11 05 am" src="https://user-images.githubusercontent.com/33464354/54030896-0c9e6200-417b-11e9-921f-792091c821a3.png">

Presumably, the following is the same model, but expressed using the excel column names:

```
EffCompRate ~ C + NotCompRate(-1) + OneBidRate + SdbRate
```

**Comment by csmcallister** [comment]
closed by 4febef7
